src/lib/Projection/Versor.ts

```diff
@@ -0,0 +1,64 @@
+export class Versor {
+  static fromAngles([l, p, g]: [number, number, number]): [number, number, number, number] {
+    l *= Math.PI / 360;
+    p *= Math.PI / 360;
+    g *= Math.PI / 360;
+    const sl = Math.sin(l),
+      cl = Math.cos(l);
+    const sp = Math.sin(p),
+      cp = Math.cos(p);
+    const sg = Math.sin(g),
+      cg = Math.cos(g);
+    return [
+      cl * cp * cg + sl * sp * sg,
+      sl * cp * cg - cl * sp * sg,
+      cl * sp * cg + sl * cp * sg,
+      cl * cp * sg - sl * sp * cg
```

Comment:
These should be something like `const sineOf1 =...`, `const cosineOfP = ...` etc.

Reply:
That seems fine to me. Might just worth having it as a general rule of thumb to avoid abbreviated variable names/shortened variable names where possible. Even something like ` x0 `  or  ` x1 `  should likely be  ` xCoordinateZero `  ,  ` xCoordinateOne`, etc.  This is especially useful/important if we start needing things like x_0 ("x naught" with the 0 being a subscript) and makes it so it's clearer when something is a coordinate value versus some sort of "functional zeroing"